### PR TITLE
Updated referral email text

### PR DESCRIPTION
**Pull request by dylanlott**
Change referral email text to remote dollar amounts and match other changes.

### CODE REVIEW

File: templates/referral.html [hunk 1]
```diff
@@ -161,29 +161,23 @@
...
 
-                                                    <p>
-                                                        {{#if url}}
-                                                            <a href="{{url}}">{{url}}</a>
-                                                        {{/if}}
-                                                    </p>
+
+                                                    <p>Claim your free credits today and experience the cloud in a new way.</p>
+
+                                                    <p>Sign up below and get 50GB of storage and 50GB of transfer per month (for the next three months)-- on us!</p>
```
Comment (barbaraliau) [inline templates/referral.html]: I told you wrong. Should be 'get 50GB of storage and 50GB of bandwidth'. Bandwidth, not transfer.

File: templates/referral.txt [hunk 2]
```diff
@@ -7,7 +7,7 @@ Storj is a decentralized cloud storage platform. It’s a blazing fast, highly s
 
 Claim your free credits today and experience the cloud in a new way.
 
-Get 50Gb Free, just follow the link below:
+Sign up with the link below and get 50GB of storage and 50GB of transfer per month (for the next three months)-- on us!
```
Comment (barbaraliau) [inline templates/referral.txt]: Same here.

File: templates/referral.html (hunk 1 above)
Comment (dylanlott) [reply templates/referral.html]: Fixed

**Comment by phutchins** [comment]
LGTM, Merging.

**Pull request merged**